pettingzoo/utils/wrappers/base.py

```diff
@@ -60,7 +64,7 @@ def observation_spaces(self):
             ) from e
 
     @property
-    def action_spaces(self):
+    def action_spaces(self) -> dict[str, gymnasium.spaces.Space]:
```

Comment:
We should probably check if gymnasium has this removed yet and if so we remove it too

Reply:
Ah right. Well in a few other files we have already explicitly deprecated the use of env.action_spaces(), so we might want to make a deprecation note here or remove it altogether.

References: https://github.com/Farama-Foundation/PettingZoo/blob/a825e94262eef7b1cb617bb54abfca3df68d0acf/pettingzoo/utils/conversions.py#L128
https://github.com/Farama-Foundation/PettingZoo/blob/a825e94262eef7b1cb617bb54abfca3df68d0acf/pettingzoo/utils/wrappers/base.py#L50
https://github.com/Farama-Foundation/PettingZoo/blob/a825e94262eef7b1cb617bb54abfca3df68d0acf/pettingzoo/utils/wrappers/base_parallel.py#L50

Reply:
There is already a deprecation warning for action spaces

https://github.com/Farama-Foundation/PettingZoo/blob/a825e94262eef7b1cb617bb54abfca3df68d0acf/pettingzoo/utils/wrappers/base.py#L64